web/js/map/wv.map.datelinebuilder.js

```diff
@@ -1,13 +1,15 @@
-var wv = wv || {};
-wv.map = wv.map || {};
+import React from 'react';
+import ReactDOM from 'React-dom';
+import WVC from 'worldview-components';
```

Comment:
Do we need to import the entire `WVC`? Or can you do something like `import DateLine, LineText from 'worldview-components';`

Reply:
Then change these two lines:
```
lineFactory = React.createFactory(WVC.DateLine);
textFactory = React.createFactory(WVC.LineText);
```
To drop the `WVC.` prefix